metricbeat/Jenkinsfile.yml

```diff
@@ -75,3 +75,8 @@ stages:
         mage: "mage build unitTest"
         platforms:             ## override default labels in this specific stage.
             - "windows-7-32-bit"
+    packaging-linux:
+        packaging-linux: "mage package"
+        e2e:
+            enabled: true
+            entrypoint: 'metricbeat-test.sh'
```

Comment:
This is already implemented with:
- https://github.com/elastic/e2e-testing/pull/705